reference/architecture/design/l3-interconnect-fabric.md

```diff
@@ -122,7 +122,7 @@ The two methods are:
     [leaf/spine](http://bradhedlund.com/2012/10/24/video-a-basic-introduction-to-the-leafspine-data-center-networking-fabric-design/){:target="_blank"}
     architecture, or via a set of spine switches, each of which is also
     a unique AS. We'll refer to this as the *AS per rack* model. This
-    model is detailed in [IETF RFC 7938](https://tools.ietf.org/search/rfc7938){:target="_blank"}.
```

Comment:
This change looks wrong - could you revert it? It's meant to link to a specific RFC.

Reply:
Reverted.